### ISSUE TITLE
Fix for search not working

### ISSUE THREAD

**Issue by AndrewMcAlister** [notepad-plus-plus/notepad-plus-plus#3611]
To make search work.
Uninstall AND IMPORTANTLY choose the delete settings/preferences option
Then re-install

**Comment by dail8859** [comment]
If you have an issue, then report the issue. Providing a "fix" for some unknown bug is not helpful.

**Comment by AndrewMcAlister** [comment]
I was doing you guys a favour because i found the bug had been reported 58 times without resolution. Now you know it has something to do with user settings.   I tried  to link it to the bug, but the login system took me away from where i was looking.  I wasted my time at work helping you guys out when i was actually very busy.  So the tone of your email is not appreciated.


Andrew

On 10 Aug. 2017 3:02 pm, "dail8859" <notifications@github.com<mailto:notifications@github.com>> wrote:

Closed #3611<https://github.com/notepad-plus-plus/notepad-plus-plus/issues/3611>.
